functions/src/resolve-market.ts

```diff
@@ -202,18 +201,10 @@ export const resolveMarket = async (
       loanPayouts
     )
 
-  const userCount = uniqBy(payouts, 'userId').length
+  // Should we combine all the payouts into one txn?
   const contractDoc = firestore.doc(`contracts/${contractId}`)
-
-  if (userCount <= 499) {
-    await firestore.runTransaction(async (transaction) => {
-      payUsers(transaction, payouts)
-      transaction.update(contractDoc, contract)
-    })
-  } else {
-    await payUsersMultipleTransactions(payouts)
-    await contractDoc.update(contract)
-  }
+  await payUsersTransactions(payouts, contractId)
+  await contractDoc.update(contract)
```

Comment:
A nice feature of the previous code was that it tried to update the contract in the same transaction that pays out users. So if paying one user failed or writing  the contract failed, it would not do any update. 

But I think this is fine, since it's simpler, and it didn't work for more than 500 uses anyway.